online.codingblocks.com/components/AboutCourse/IntroductionCard.vue

```diff
@@ -4,21 +4,27 @@
       <div class="border-card white">
         <img class="back-img" :src="course['cover-image']" alt="" />
 
-        <div class="row justify-content-between align-items-center position-relative no-gutters">
+        <div
+          class="row justify-content-between align-items-center position-relative no-gutters"
+        >
           <div class="col-lg-10 col-sm-9 col-8">
             <div>
-              <span class="bold font-xl">{{course.title}}</span>
+              <span class="bold font-xl">{{ course.title }}</span>
               <span class="ml-4">
-                <a href="#" class="white">
-                  <i class="far fa-lg fa-heart"></i>
-                </a>
+                <button id="wishlist_btn" @click="addToWishlists()">
+                  <i
+                    class="fa-heart fa-lg"
+                    v-bind:class="{ far: unfilled, fas: filled }"
```

Comment:
`<div v-bind:class="[isActive ? activeClass : '', errorClass]"></div>`
this is much better